documentation/docs/30-advanced/30-link-options.md

```diff
@@ -122,8 +122,8 @@ To disable any of these options inside an element where they have been enabled,
 </div>
 ```
 
-To apply an attribute to an element conditionally, do this (`"true"` and `"false"` are both accepted values):
+To apply an attribute to an element conditionally, do this:
 
-```html
-<div data-sveltekit-reload={shouldReload}>
+```svelte
+<div data-sveltekit-preload-data={condition ? "hover" : "false"}>
```

Comment:
I would expect we wouldn't have to use quotes with a boolean value. I see @Rich-Harris used quotes when discussing it earlier, which is probably why you did it that way, but I'm sort of assuming that was just because of its usage as an HTML attribute value like `<div data-sveltekit-preload-data="false">`
```suggestion
<div data-sveltekit-preload-data={condition ? 'hover' : false}>
```